pkg/util/operatorconfig/modules.go

```diff
@@ -0,0 +1,59 @@
+package operatorconfig
```

Comment:
`componentconfig`? We are not directly configuring the Operator, but configuring what components can be enabled.

Reply:
I do not have the strongest opinion here, but I like `componentconfig` more tbh.

Reply:
it is a config for the operator in general, to decide which components are "available"

you could argue that we only provide it to the webhook currently, so `webhookconfig`, but that may change in the future

Another I thought about is `installinfo`/`installconfig` as it is information related to the installation of the operator

`componentconfig` tells me nothing
- which component is the config for?

Reply:
`operatorcomponentconfig` probably is too long right?

`installconfig` sounds good in my opinion.

Reply:
renamed it to`installconfig`